R/read_company_data.R

```diff
@@ -1,49 +1,42 @@
 #' Read in company finacial data processed from eikon exports and AR master data
 #' that contain information on multiple credit risk inputs and company
-#' production plans.
+#' production plans, aggregated to the ticker/technology/year level for
+#' `asset_type` bonds and aggregated to the company/technology/year level for
+#' `asset_type` equity.
 #'
 #' @param path A string that points to the location of the file containing the
 #'   company financial data.
+#' @param asset_type A string indicating if company data are for analysis for
+#'   bond or equity.
 #'
 #' @family import functions
 #'
 #' @export
-read_company_data <- function(path = NULL) {
+read_company_data <- function(path = NULL, asset_type) {
+
   path %||% stop("Must provide 'path'")
 
+  if (!asset_type %in% c("bonds", "equity")) {
```

Comment:
should this at some point include loans as a valid asset type? should we then have another place that defines valid asset types?

Reply:
see below